### ISSUE TITLE
Instructions didnt work on Ubuntu 16.04

### ISSUE THREAD

**Issue by duncs** [opsview/application-docker#1]
Using the instructions to modify `/lib/systemd/system/docker.service` on Ubuntu 16.04 and amending the config line `ExecStart=/usr/bin/docker -d -H 127.0.0.1:4243 -H unix:///var/run/docker.sock` caused docker daemon not to start.

Instead, need to use `sudo systemctl edit docker` and amend the override file as follows:
{code}
[Service]
ExecStart=/usr/bin/dockerd -H 127.0.0.1:4243 -H unix:///var/run/docker.sock```
{code}

basically using `/usr/bin/dockerd` instead of `/usr/bin/docker -d`.  Using the override file in this way should also survive any upgrades.

**Comment by andymadge** [comment]
The override needs an empty ExecStart line, like this:

```
[Service]
ExecStart=
ExecStart=/usr/bin/dockerd -H 127.0.0.1:4243 -H unix:///var/run/docker.sock
```

ExecStart in `override.conf` is additive, so it adds to the existing value.  You need to clear it first, then set what you need

**Comment by cathalo169** [comment]
I am not having success with this at all. The host shows 127.0.0.1:4243 using netstat -tuna, but no go on the monitoring tool. I've even tried 0.0.0.0:4243. Is there something that needs to be done to the container. I am not a docker person, so learning on the fly.  Please send me a message if you think you can help. Cheers
